roles/backend_setup/tasks/cache_setup.yml

```diff
@@ -6,6 +6,14 @@
 # Extend the existing volume group with the SSD (assuming SSD is used for
 # caching)
 
+# Fail if the cache disk is not present
+- name: Check if the cachedisk exists, else fail
```

Comment:
can we move all such validations to a pre-checks dir? And execute the pre-checks in the main.yml?

Reply:
We can do that, is there a possibility that RHHI would be installed without cache? If so we have to maintain the cachedisk check separate and call only when user calls for setting up cache.

Reply:
Yes, it's possible as cache setup is optional

Reply:
> Yes, it's possible as cache setup is optional

@sabose I'm sorry I missed this. So, if cache is optional, we need not move it to main.yml and call this every time irrespective of whether user wants to setup cache or not. Over here it gets validated only if user tries to setup cache.